docs/parameters.md

```diff
@@ -0,0 +1,713 @@
+# nf-core/dualrnaseq: Parameters
```

Comment:
These could all be added to the JSON schema & the description of each copied into the JSON schema. That would also allow you to launch the pipeline using https://nf-co.re/launch 🎉

Reply:
All done - didn't know about that launcher, a cool idea!